src/config/options.schema.json

```diff
@@ -970,6 +977,18 @@
                 "Do widening on contexts. Keeps a map of function to call state; enter will then return the widened local state for recursive calls.",
               "type": "boolean",
               "default": false
+            },
+            "gas_value": {
+              "title": "ana.context.gas_value",
+              "description": "x denotes the gas value for the ContextGasLifter. Negative value means deactivated, zero means context-insensitve analysis. If enabled, the first x recursive calls of the call stack are analyzed context-sensitive, the remaining with no context.",
+              "type": "integer",
+              "default": -1
+            },
+            "callString_length": {
+              "title": "ana.context.callString_length",
+              "description": "Length of the call string that should be used as context for the Call String analysis. In case the value is zero, the analysis is context-insensitive. For a negative value an infinite call string is used!",
+              "type": "integer",
+              "default": 2
```

Comment:
Does this change our default behavior? or does some other analysis need to be activated for this to take effect? If yes, it should be mentioned here.